native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -128,11 +128,19 @@ const modifiers = {
     container: {
       width: tokens.spacingXl,
     },
+    button: {
+      paddingLeft: 0,
```

Comment:
`paddingHorizontal` could be used here right?

Reply:
Didn't know about that prop, thanks!